website/cue/reference/components/sources/azure_event_hubs.cue

```diff
@@ -0,0 +1,102 @@
+package metadata
+
+components: sources: azure_event_hubs: {
+	title: "Azure Event Hubs"
+
+	classes: {
+		commonly_used: false
+		delivery:      "at_least_once"
+		deployment_roles: ["daemon", "sidecar"]
+		development:   "beta"
+		egress_method: "batch"
+		stateful:      false
+	}
+
+	features: {
+		acknowledgements: false
+		collect: from: {
+			service: services.kafka
+			interface: {
+				socket: {
+					api: {
+						title: "Kafka protocol"
+						url:   urls.kafka_protocol
+					}
+					direction: "incoming"
+					port:      9093
+					protocols: ["tcp"]
+					ssl: "optional"
+				}
+			}
+		}
+	}
+
+	support: {
+		requirements: []
+		notices: []
+		warnings: []
+	}
+
+	configuration: {
+		connection_string: {
+			description: """
+				The connection string.
+				See [here](\(urls.azure_event_hubs_connection)) for details.
+				"""
+			required:    true
+			type: string: {
+				examples: ["Endpoint=sb://mynamespace.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=XXXXXXXXXXXXXXXX"]
+			}
+		}
+		namespace: {
+			common:      false
+			description: "The namespace name."
+			required:    true
+			type: string: {
+				examples: ["namespace"]
+				options: {}
+			}
+		}
+		queue_name: {
+			common:      false
+			description: "The name of the queue to listen to."
+			required:    true
+			type: string: {
+				examples: ["queue"]
+			}
+		}
+		group_id: {
+			common:      false
+			description: "The name of the consumer group."
+			required:    false
+			type: string: {
+				default: "$DEFAULT"
+				examples: ["GROUP"]
+			}
+		}
+	}
+
+	how_it_works: {
+		kafka: {
+			title: "kafka"
+			body:  """
+                This component is a simple wrapper over the `kafka` source.
+                See the documentation [here](\(urls.azure_event_hubs_kafka_ecosystem))
+                for details on how `azure_event_hubs` can use `kafka`.
```

Comment:
I'd rather have this say something like "this component leverages event hubs' compatibility with kafka and requires <whatever account level>..."